homeassistant/components/airzone/entity.py

```diff
@@ -156,7 +157,7 @@ def __init__(
             identifiers={(DOMAIN, f"{entry.entry_id}_{system_zone_id}")},
             manufacturer=MANUFACTURER,
             model=self.get_airzone_value(AZD_THERMOSTAT_MODEL),
-            name=f"Airzone [{system_zone_id}] {zone_data[AZD_NAME]}",
```

Comment:
I'd prefer to use something like f"Zone [{system_zone_id}] {zone_data[AZD_NAME]}" if we really need to drop the "Airzone" prefix here.